data/json/effects.json

```diff
@@ -2954,6 +2954,18 @@
     "//": "No morale_mod is currently possible in effects, for some reason.  As soon as it's implemented, it goes here.",
     "rating": "bad"
   },
+  {
+    "type": "effect_type",
+    "id": "genetics_damaged",
+    "name": [ "Mild Genetic Damage", "Genetic Damage", "Severe Genetic Damage" ],
+    "desc": [
+      "After some mutations, you feel slightly uncomfortable - and that's it.",
```

Comment:
I'd suggest "After mutating somewhat, you're feeling slightly uncomfortable, although it's nothing major."